### PR TITLE
FreeSoftwareFoundation.LameXP-AudioEncoderFront-End version 4.1.8.1

### PR DESCRIPTION
**Pull request by max-stockhausen**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16326)

### CODE REVIEW

File: manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: FreeSoftwareFoundation.LameXP-AudioEncoderFront-End
+PackageVersion: 4.1.8.1
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://downloads.sourceforge.net/project/lamexp/LameXP.2019-12-18.Release-Static.Build-2240.exe?ts=gAAAAABgu28WYlvGRxhpwwLCHinlgBfFLOHqO5gx1fbF_aB9PeV6rSar51qI83RILBz9u7F0UAjWUGw9tor3dClFWj6Jc6vb1A%3D%3D&r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Flamexp%2Ffiles%2Flatest%2Fdownload
```
Comment (ItzLevvie) [inline manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.installer.yaml:11]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/lamexp/files/LameXP.2019-12-18.Release-Static.Build-2240.exe/download
```

File: manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FreeSoftwareFoundation.LameXP-AudioEncoderFront-End
+PackageVersion: 4.1.8.1
```
Comment (ItzLevvie) [inline manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 4.18 Final-1 [Build #2240]
```

File: manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: FreeSoftwareFoundation.LameXP-AudioEncoderFront-End
+PackageVersion: 4.1.8.1
```
Comment (ItzLevvie) [inline manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.yaml:4]:
```suggestion
PackageVersion: 4.18 Final-1 [Build #2240]
```

File: manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: FreeSoftwareFoundation.LameXP-AudioEncoderFront-End
+PackageVersion: 4.1.8.1
```
Comment (ItzLevvie) [inline manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.installer.yaml:4]:
```suggestion
PackageVersion: 4.18 Final-1 [Build #2240]
```

File: manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FreeSoftwareFoundation.LameXP-AudioEncoderFront-End
+PackageVersion: 4.1.8.1
+PackageLocale: en-US
+Publisher: Free Software Foundation
```
Comment (ItzLevvie) [inline manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.locale.en-US.yaml:6]:
```suggestion
Publisher: LoRd_MuldeR <mulder2@gmx.de>
```

File: manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FreeSoftwareFoundation.LameXP-AudioEncoderFront-End
+PackageVersion: 4.1.8.1
+PackageLocale: en-US
+Publisher: Free Software Foundation
+PackageName: LameXP - Audio Encoder Front-End
```
Comment (ItzLevvie) [inline manifests/f/FreeSoftwareFoundation/LameXP-AudioEncoderFront-End/4.1.8.1/FreeSoftwareFoundation.LameXP-AudioEncoderFront-End.locale.en-US.yaml:7]:
```suggestion
PackageName: LameXP
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Comment by max-stockhausen** [comment]
Can I accept all those changes at once?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to do it individually; there should be a button called "Commit suggestion" or something along the lines, or otherwise, you can use the GitHub Desktop client on Windows to modify the files manually and push the changes to your branch which will then update the PR with those changes.

**Comment by max-stockhausen** [comment]
Because I cant commit to this Branch I created a new request: #16957

**Pull request closed without merging**